system/autoware_mrm_handler/src/mrm_handler/mrm_handler_core.cpp

```diff
@@ -586,6 +577,33 @@ bool MrmHandler::isArrivedAtGoal()
   return state->mode == OperationModeState::STOP;
 }
 
+bool MrmHandler::isAvailableCurrentOperationMode()
```

Comment:
`isHoge` like function name sounds begin `const`, so updating global variable inside is strange.

Reply:
ok